tests/ha/test_server_pod_failure.py

```diff
@@ -517,9 +494,9 @@ def test_writes_during_server_pod_down(self):
         self.test_prefix = 'test-39907-1'
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                     log_prefix=self.test_prefix, skipcleanup=True,
-                                                    nsamples=2, nclients=2)
+                                                    nsamples=2, nclients=2, setup_s3bench=False)
```

Comment:
Do we need this here?

Reply:
Yes. event_s3_operation call at line 446 will install the s3bench. Why do we need to re-install the same